src/lib/ngsi/NotifyCondition.h

```diff
@@ -46,6 +46,12 @@ typedef struct NotifyCondition
   std::string               type;            // Mandatory
   ConditionValueList        condValueList;   // Optional
   RestrictionString         restriction;     // Optional
```

Comment:
For documentation, I'd suggest to mark `expresion` as a field that is only used in NGISv1, e.g. `// Only used by NGSIv2 subscription`